packages/addon-dev/src/rollup-gjs-plugin.ts

```diff
@@ -19,12 +22,17 @@ export default function rollupGjsPlugin(
         return null;
       }
       let input = readFileSync(id, 'utf8');
-      let code = processor.process(input, {
+      let codeWithInlineMap = processor.process(input, {
         filename: id,
         inline_source_map,
       });
+
+      let map = convert.fromSource(codeWithInlineMap).toJSON();
+      let code = codeWithInlineMap.replace(/\/\/# sourceMappingURL=\/.+$/, '');
+
```

Comment:
Best to start with `^` too in the regex.
Still would not protect against this if this text is inside a a template literal.

Reply:
true -- it _is_ supposed to be the last line, so I could do a `.lastIndexOf` sort of thing